app/src/org/commcare/connect/database/ConnectDatabaseHelper.java

```diff
@@ -39,21 +30,8 @@ public class ConnectDatabaseHelper {
     public static IDatabase connectDatabase;
     static boolean dbBroken = false;
 
-    public static void handleReceivedDbPassphrase(Context context, String remotePassphrase) {
-        ConnectDatabaseUtils.storeConnectDbPassphrase(context, remotePassphrase, false);
-        try {
-            //Rekey the DB if the remote passphrase is different than local
-            String localPassphrase = ConnectDatabaseUtils.getConnectDbEncodedPassphrase(context, true);
-            if (connectDatabase != null && connectDatabase.isOpen() && !remotePassphrase.equals(localPassphrase)) {
-                DatabaseConnectOpenHelper.rekeyDB(connectDatabase, remotePassphrase);
```

Comment:
can we also clear out the `rekeyDB` from `DatabaseConnectOpenHelper` or is it still used elsewhere ?